ClockDisplay.java

```diff
@@ -78,7 +78,23 @@ public String getTime()
      */
     private void updateDisplay()
     {
-        displayString = hours.getDisplayValue() + ":" + 
+        
+        String hour12;
+        hour_value=hours.getValue();
```

Comment:
you should create a String named meridian and set it to "AM" here. THen if hours >=12, set it to PM in your test